Fix issue #619 - Simplify usage of PrivateKeyJwtCredentials for event notification (#633)
camaraproject/Commonalities: documentation/CAMARA-API-Event-Subscription-and-Notification-Guide.md

The JWK Set ((JSON Web Key Set)) contains the cryptographic keys used for authentication ([RFC 7517](https://www.rfc-editor.org/rfc/rfc7517.html)) 

> This information could be shared out of band (manual exchange, contract data, via the Operate API, etc.) or as input using PrivateKeyJWTCredential credentials.

Reviewer: I propose to rephrase it (as the schema name `PrivateKeyJWTCredential` is not mentioned in the document and the exact name is not mandatory) :
```suggestion
This information could be shared out of band (manual exchange, contract data, via the Operate API, etc.) or as properties of `sinkCredential` object in subscription request/response.
```

    end
  end

>   opt Using PRIVATE_KEY_JWT sink credential type (credentials exchanged out of bound)

Reviewer: Typo?
```suggestion
  opt Using PRIVATE_KEY_JWT sink credential type (credentials exchanged out of band)
```

      cspres -->> appclient: Invalid sink credential type or missing pre-shared information 
      end
    end
  end

>   opt Using PRIVATE_KEY_JWT sink credential type (credentials exchanged in bound)

Reviewer: Small correction:
```suggestion
  opt Using PRIVATE_KEY_JWT sink credential attributes (credentials exchanged in band)
```

>   opt Using PRIVATE_KEY_JWT sink credential type (credentials exchanged in bound)

Reviewer: Small correction:
```suggestion
  opt Using PRIVATE_KEY_JWT sink credential attributes (credentials exchanged in band)
```

    note over appclient,cspclient: Onboarding: <br> API Consumer shares nothing
    opt Subscription

>     appclient ->> cspres: POST /subscriptions<br>Body: {<br>sink: <API Consumer 2>/sink,<br>sinkCredential:{ credentialType: PRIVATE_KEY_JWT,<br> tokenUri:<API Consumer 2>/tokenUri,,<br> clientId:<API Provider clientId> },<br>...}

Reviewer 2: ```suggestion
    appclient ->> cspres: POST /subscriptions<br>Body: {<br>sink: <API Consumer 2>/sink,<br>sinkCredential:{ credentialType: PRIVATE_KEY_JWT,<br> tokenUri:<API Consumer 2>/tokenUri,<br> clientId:<API Provider clientId> },<br>...}
```

Reviewer 2: minor editorial
